test/integration/upgrade_test.go

```diff
@@ -51,10 +51,10 @@ func voteOnProposal(m testCommon.TestConfig, proposalId uint64) {
 	}
 }
 
-// propose an upgrade to the v0.4.0 software version
+// propose an upgrade to the v0.4.1 software version
 func proposeUpgrade(m testCommon.TestConfig) (proposalId uint64, proposalHeight int64) {
 	ctx := context.Background()
-	name := v0_4_0.UpgradeName
+	name := v0_4_1.UpgradeName
```

Comment:
needs to be v0.5.0

Reply:
https://github.com/allora-network/allora-chain/pull/594